src/web/app/components/question-submission-form/question-submission-form.component.html

```diff
@@ -1,4 +1,4 @@
-<div class="card margin-top-30px" xmlns="http://www.w3.org/1999/html">
+<div class="card margin-top-30px" id="{{isModeratedQuestion ? 'moderated-question': ''}}" xmlns="http://www.w3.org/1999/html">
```

Comment:
This is how scrolling was implemented in v6

Reply:
It is useful to look at how thing get implemented. However, Angular introduces a separation between logic and DOM manipulation. So it usually not recommended to manipulate HTML DOM element in Angular code (i.e. Let the dirty work done by the Angular framework itself). In addition, even if you want to do this, why not just do `[id]="question.questionId"` instead of doing this hacky `[id]="moderated-question"` ?

Reply:
Sure